src/relative-time.js

```diff
@@ -62,31 +64,29 @@ export default class RelativeTime {
     const month = Math.round(day / 30)
     const year = Math.round(month / 12)
     if (ms < 0) {
-      return 'just now'
+      return rtf.format(0, 'second')
```

Comment:
```js
new Intl.RelativeTimeFormat('en', {numeric: 'auto'}).format(0, 'second')
//=> "now"
```

Reply:
Nice! I looked at the list [here](https://github.com/tc39/proposal-intl-relative-time#example-1) and didn't find a substitute to "Just now" so I wasn't sure how we were going to solve that 😄 

https://github.com/tc39/proposal-intl-relative-time/pull/107